src/components/Link/Link.module.css

```diff
@@ -0,0 +1,13 @@
+@import '../../design-tokens/mixins.css';
+
+/*------------------------------------*\
+    # LINK
+\*------------------------------------*/
+
+/**
+ * Link button with link variant styles
+ * 1) Override display: inline-flex from ClickableStyle
+ */
+.link--link {
+  display: inline; /* 1 */
+}
```

Comment:
This is a little awkward but I only want to add a style to the `Link` component when it's using the "link" variant.